diff -ru -x CVS xmlcatmgr/catalogs.mk ../lzt/xmlcatmgr/catalogs.mk
--- xmlcatmgr/catalogs.mk	2015-11-25 10:53:42.000000000 -0200
+++ ../lzt/xmlcatmgr/catalogs.mk	2017-04-16 12:28:00.000000000 -0300
@@ -32,14 +32,14 @@
 # Convert SGML_CATALOGS files into arguments for SGML_ENTRIES.
 .if !empty(SGML_CATALOGS)
 .  for c in ${SGML_CATALOGS}
-SGML_ENTRIES+=	CATALOG ${PREFIX:=${c}} --
+SGML_ENTRIES+=	CATALOG ${c} --
 .  endfor
 .endif
 
 # Convert XML_CATALOGS files into arguments for XML_ENTRIES.
 .if !empty(XML_CATALOGS)
 .  for c in ${XML_CATALOGS}
-XML_ENTRIES+=	nextCatalog ${PREFIX:=${c}} --
+XML_ENTRIES+=	nextCatalog ${c} --
 .  endfor
 .endif
 
